src/components/services/UptimeKuma.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+  <Generic :item="item">
+    <template #content>
+      <p class="title is-4">{{ item.name }}</p>
+      <p class="subtitle is-6">
+        <template v-if="item.subtitle">
+          {{ item.subtitle }}
+        </template>
+        <template v-else-if="status">
+          {{ statusMessage }}
+        </template>
+      </p>
+    </template>
+    <template #indicator>
+      <div v-if="status" class="status" :class="status">
+        {{ uptime }}&percnt;
+      </div>
+    </template>
+  </Generic>
+</template>
+
+<script>
+import service from "@/mixins/service.js";
+import Generic from "./Generic.vue";
+
+export default {
+  name: "UptimeKuma",
+  mixins: [service],
+  props: {
+    item: Object,
+  },
+  components: {
+    Generic,
+  },
+  data: () => ({
+    incident: null,
+    heartbeat: null,
+  }),
+  computed: {
+    dashboard: function () {
+      return this.item.slug ? this.item.slug : "default";
+    },
+    status: function () {
+      if (!this.incident) {
+        return "";
+      }
+      return this.incident.incident == null ? this.pageStatus : "bad";
+    },
+    lastHeartBeatList: function () {
+      let result = {};
+
+      for (let id in this.heartbeat.heartbeatList) {
+        let index = this.heartbeat.heartbeatList[id].length - 1;
+        result[id] = this.heartbeat.heartbeatList[id][index];
+      }
+
+      return result;
+    },
+    pageStatus: function () {
+      if (!this.heartbeat) {
+        return "";
+      }
+      if (Object.keys(this.heartbeat.heartbeatList).length === 0) {
+        return "";
+      }
+      let result = "good";
+      let hasUp = false;
+      for (let id in this.lastHeartBeatList) {
+        let beat = this.lastHeartBeatList[id];
+        console.log(beat.status);
```

Comment:
```suggestion
```

Remove console.log

Reply:
Haha, my bad :) Troubleshooting the API response there